chipy_org/apps/sponsors/models.py

```diff
@@ -21,8 +22,8 @@ class Meta:
 
 
 class GeneralSponsor(models.Model):
-    sponsor = models.ForeignKey("sponsors.Sponsor")
-    about = models.TextField("About this sponsorship", blank=True, null=True)
+    sponsor = models.ForeignKey("sponsors.Sponsor", on_delete=models.CASCADE)
+    about = models.TextField(u"About this sponsorship", blank=True, null=True)
     about_short = models.CharField(
         "Brief description of sponsorship", max_length=128, blank=True, null=True
```

Comment:
A similar thing to the above comment. The 'about' TextField here is a unicode string, while the 'about_short' CharField is not.

Reply:
Wouldn't the 'u' be unnecesary in the 'about' field because, since Python 3.0, 'str' are represented as unicode already? I explain more about it in a previous comment. We are using python 3.6.9 .

Reply:
Actually we're upgrading to 3.8.2 😄 
And yes we can start removing those.